docs/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,33 @@
+* xref:index.adoc[]
+* xref:installation.adoc[]
+* xref:pages.adoc[]
+** xref:page-attributes.adoc[]
+** xref:layout.adoc[]
+** xref:disabling-pub.adoc[]
+** xref:implicit-page-vars.adoc[]
+** xref:doctitle.adoc[]
+* xref:blog.adoc[]
+** xref:classifying.adoc[]
+** xref:time.adoc[]
+** xref:drafts.adoc[]
+* xref:liquid.adoc[]
+* xref:excerpts.adoc[]
+* xref:build.adoc[]
+* xref:configuration.adoc[]
+* xref:asciidoctor.adoc[]
+** xref:base-dir.adoc[]
+* xref:liquid-attributes.adoc[]
+* xref:hard-line-breaks.adoc[]
```

Comment:
I think hard-line-breaks.adoc should be nested under Configuration (as it is directly related to configuration, but still deserves its own page).